### DISCUSSION TITLE
Submit photo of ID card for teacher account

### DISCUSSION THREAD

**Discussion by ali-nayeem** [community/community#145013] [GitHub Education]
### Hi everyone,

I need to submit my ID card to get the education subscription. But I only see that I need to take picture using my laptop camera and no way to upload images. I need to submit multiple images. Please I need help in this regard.

**Comment by FeiSuoerDe** [comment]
The whole community has this problem，Please upload a document with the date of admission and graduation and try multiple times

**Comment by mrrizalaziz** [comment]
yeah how to get the teacher's account?
